third_party/opa/data-and-policies/user-created-policy-3/sample_policies.rego

```diff
@@ -0,0 +1,87 @@
+package dataapi.authz
+
+verdict[output] {
+	count(rule) == 0
+	output = {"action": {"name":"Deny"}, "policy": "Deny by default"}
+}
+
+verdict[output] {
+	count(rule) > 0
+	output.action.name == "Deny"
+	output = rule[_]
+}
+
+verdict[outputFormatted] {
+	count(rule) > 0
+	output.action.name != "Deny"
+	output = rule[_]
+	outputFormatted := {"action": {"name":output.action.name, output.action.name: output.action.columns}, "policy": output.policy}
+}
+
+rule[{"action": {"name":"RedactColumn", "columns": column_names}, "policy": description}] {
+	description := "Columns with Confidential tag to be redacted before read action"
```

Comment:
What does the policy description should say?
1. Describe the action that needs to be done. i.e., redact column from the output
2. Describe the "reason" or policy logic that was evaluated, i.e., redact columns with confidential tag for a data scientist running fraud detection.
Which should it be?
the description in the policies are not consistent in their expressivness.

Reply:
It should be 2

Reply:
Fixed the description of the rules. These are sample rules for demonstrations.